packages/flutter/lib/src/services/platform_messages.dart

```diff
@@ -31,6 +31,9 @@ import 'platform_channel.dart';
 class BinaryMessages {
   BinaryMessages._();
 
+  /// The messenger which sends the bytes for this channel, not null.
+  static final BinaryMessenger binaryMessenger = ServicesBinding.instance.provideBinaryMessenger();
```

Comment:
if it's for "this" channel, it shouldn't be static.

Reply:
oh never mind, i see, these are all static. hm.

Reply:
Adjusted the text a bit.